### PR TITLE
Match `simple-shared-worker/multiply.js` to use same event-handler loop as `simple-web-worker/main.js`

### PR DESCRIPTION
**Pull request by Marcial1234**
Caught this small inconsistency - on one SW each event handler is attached separately, in the other they're done inside a loop. 
Less code seems better.

[See local reference](../blob/f3e0dbb/web-workers/simple-web-worker/main.js#L9-L14)

Perhaps the [external reference](https://github.com/mdn/content/blob/f98deaa21bc31b006cbb597309d3aef7843c4b28/files/en-us/web/api/web_workers_api/using_web_workers/index.md?plain=1#L68-L78) could be reduced/simplified as well?

### CODE REVIEW

**Review by king-2929 — APPROVED** [review]

**Review by king-2929 — APPROVED** [review]

**Review by king-2929 — APPROVED** [review]

**Review by king-2929 — APPROVED** [review]

**Comment by bsmth** [comment]
Thanks a lot for opening this one. Related PR landed similar fixes in:

- [x] https://github.com/mdn/dom-examples/pull/170

**Comment by Marcial1234** [comment]
Sure!

**Comment by Marcial1234** [comment]
https://github.com/mdn/content/pull/37208 @bsmth

File: web-workers/simple-shared-worker/multiply.js [hunk 1]
```diff
@@ -6,15 +6,12 @@ const result1 = document.querySelector(".result1");
...
-    myWorker.port.postMessage([first.value, second.value]);
-    console.log("Message posted to worker");
-  };
-
-  second.onchange = function () {
-    myWorker.port.postMessage([first.value, second.value]);
-    console.log("Message posted to worker");
-  };
+  [first, second].forEach(input => {
+    input.onchange = function() {
```
Comment (bsmth) [inline web-workers/simple-shared-worker/multiply.js:10]:
```suggestion
    input.onchange = () => {
```

File: web-workers/simple-shared-worker/multiply.js [hunk 2]
```diff
@@ -6,15 +6,12 @@ const result1 = document.querySelector(".result1");
...
+      myWorker.port.postMessage([first.value, second.value]);
+      console.log('Message posted to worker');
+    }
+  })
 
   myWorker.port.onmessage = function (e) {
```
Comment (bsmth) [inline web-workers/simple-shared-worker/multiply.js:16]: We can also do

```suggestion
  myWorker.port.onmessage = (e) => {
```

Comment (bsmth) [reply web-workers/simple-shared-worker/multiply.js:16]: In fact, this is already changed in the content repo, so let's make it consistent

**Review by bsmth — APPROVED** [review]
Looking great, I'm going to merge this one and the content improvements shortly! Well done and thank you!

**Pull request merged**